src/game/server/entities/plasma.cpp

```diff
@@ -0,0 +1,109 @@
+#include <engine/server.h>
```

Comment:
in pull request #77 there was a license notice, that this code was (c) Shereef Marzouk and suggested to see "licence DDRace.txt". Why there is no this notice now? Is this file written from scratch or taken from DDrace?

Reply:
I added a notice line.